pkg/controller/nodelifecycle/config/types.go

```diff
@@ -45,6 +45,6 @@ type NodeLifecycleControllerConfiguration struct {
 	// Zone is treated as unhealthy in nodeEvictionRate and secondaryNodeEvictionRate when at least
 	// unhealthyZoneThreshold (no less than 3) of Nodes in the zone are NotReady
 	UnhealthyZoneThreshold float32
-	// tenant api-server URLs
-	TenantPartitionKubeConfigs []string
+	// tenant api-server kubeconfig
```

Comment:
not very clear whether multiple TPs or single TP for this field.

Reply:
In long term, it should be single kubeconfig file, multiple TPs should be auto discovered.